tests/bmtestspolar/symfixedrankpolarfactory.m

```diff
@@ -0,0 +1,264 @@
+function M = symfixedrankpolarfactory(m, k)
+% Manifold of symmetric, positive semidefinite matrices of size m and rank k with polar quotient geometry.
+%
+% function M = symfixedrankpolarfactory(m, k)
+%
+% The first-order geometry follows the balanced quotient geometry described
+% in the paper,
+% "Linear regression under fixed-rank constraints: a Riemannian approach",
+% G. Meyer, S. Bonnabel and R. Sepulchre, ICML 2011.
+%
+% Paper link: http://www.icml-2011.org/papers/350_icmlpaper.pdf.
+%
+% The second-order geometry follows from the paper
+% "Fixed-rank matrix factorizations and Riemannian low-rank optimization",
+% B. Mishra, G. Meyer, S. Bonnabel and R. Sepulchre,
+% Computational Statistics, 29(3 - 4), pp. 591 - 621, 2014.
+%
+% A point X on the manifold is represented as a structure with two
+% fields: U and B. The matrix U (mxk) is orthonormal,
+% while the matrix B (kxk) is a symmetric, positive definite.
+%
+% Tangent vectors are represented as a structure with two fields: U and B.
+% These are the horizontal lifts of the tangent vectors on the quotient 
+% manifold.
```

Comment:
NB May 3, 2019 comment:

> Can you say more about these? I guess if V is a tangent vector at X, then V.B is symmetric, and V.U is in the tangent space to Stiefel at X.U? Or is there also a quotient thing going on, so that the tangent vector is really the horizontal lift of a tangent vector? This should be explained.

BM Sep 16, 2019 reply:

> Quotient thing is on. So, they are the horizontal lifts of the tangent vectors on the quotient space.